File: easy/tickets/tickets.java
package ejercicios;

import java.util.Scanner;

/**
 *06/05/2021
 * @author Andre Peralta
 */
public class metodo {
    public static void main(String[] args) {
        Scanner lector = new Scanner(System.in);
        
        System.out.println("Introduce el puntaje");
        int points = lector.nextInt();
        
        System.out.println("Introduce el precio de la pistola");
        int price = lector.nextInt();
        
        if(points/12 >= price){
            System.out.println("Buy it");
        }else{
            System.out.println("Try again");
        }
        
    }
    
}

File: easy/tickets/tickets.cpp
#include <iostream>

using namespace std;

int main(){

    int points;
    int cost;

    cout << "Introduce el número de puntos ganados:" << "\n";
    cin >> points;

    cout << "Introduce el precio de la pistola de agua:" << "\n";
    cin >> cost;

    if(points/12 >= cost){
        cout << "Buy it!";
    }else{
        cout << "Try again";
    }

	return 0;
}
